src/util/chunk_fs.js

```diff
@@ -55,13 +55,13 @@ class ChunkFS extends stream.Transform {
             });
             this.count = 0;
             while (chunk && chunk.length) {
-                const available_size = config.NSFS_BUF_SIZE - this.q_size;
+                const available_size = config.NSFS_BUF_SIZE_L - this.q_size;
```

Comment:
we want to avoid reading the config dynamically and instead use the largest pool.
```
const { buf_size } = get_buffers_pool(undefined);

...
const available_size = buf_size - this.q_size;
```